backend/app/models.py

```diff
@@ -44,5 +58,285 @@ class Document(models.Model):
     title = models.CharField(max_length=255, blank=True)
     word_count = models.PositiveIntegerField(blank=True, null=True, default=None)
     tokenized_text = models.JSONField(null=True, blank=True, default=None)
-    word_counts_counter = models.JSONField(null=True, blank=True, default=dict)
+    word_count_counter = models.JSONField(null=True, blank=True, default=dict)
     part_of_speech_tags = models.JSONField(null=True, blank=True, default=list)
+
+    objects = DocumentManager()
+
+    def _clean_quotes(self):
+        """
+        Scans through the text and replaces all of the smart quotes and apostrophes with their
+        "normal" ASCII variants
+
+        :param self: The Document to reformat
+        :return: A string that is identical to `text`, except with its smart quotes exchanged
+        """
+
+        smart_quotes = {
+            '“': '"',
+            '”': '"',
+            "‘": "'",
+            "’": "'",
+        }
+
+        output_text = self.text
+        for quote in smart_quotes:
+            output_text = output_text.replace(quote, smart_quotes[quote])
```

Comment:
I'm wondering about whether or not this is the ideal way to implement this quotation mark clean up. I think that lines 83-84 here will perform four passes on the input text, one for each key in `smart_quotes`, unless `.replace()` is doing something particularly smart that I don't know about. It seems like using the regex `.sub` method would implement this in one pass, and therefore more efficiently. Something like:

```
import re 

re.sub(r'[\“\”\‘\’]', '\"', self.text)
```